src/V3Options.cpp

```diff
@@ -1326,6 +1329,8 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc,
         m_hierBlocks.emplace(opt.mangledName(), opt);
     });
     DECL_OPTION("-hierarchical-child", Set, &m_hierChild);
+    DECL_OPTION("-hierarchical-params-file", CbVal,
+                [this](const char* optp) { m_hierParamsFile = optp; });
```

Comment:
Need to document this in exe_verilator as an internal flag.

Reply:
Added here: https://github.com/verilator/verilator/pull/5333/commits/d4616df37e93ad690021cce5e48a53367e97b759